wgpu/src/lib.rs

```diff
@@ -1545,8 +1545,8 @@ pub struct ComputePipelineDescriptor<'a> {
     pub layout: Option<&'a PipelineLayout>,
     /// The compiled shader module for this stage.
     pub module: &'a ShaderModule,
-    /// The name of the entry point in the compiled shader. There must be a function that returns
-    /// void with this name in the shader.
+    /// The name of the entry point in the compiled shader. There must be a function with this name
+    /// in the shader.
     pub entry_point: &'a str,
```

Comment:
I am not sure about this change because I did not test a compute shader.

Reply:
Compute shaders actually do need to return void.

Reply:
Fixed.